linux/kernel/building.md

```diff
@@ -48,32 +48,52 @@ Configure the kernel; as well as the default configuration, you may wish to [con
 <a name="default_configuration"></a>
 #### Apply the default configuration
 
-First, prepare the default configuration by running the following commands, depending on your Raspberry Pi version:
+First, prepare the default configuration by running the following commands, depending on your Raspberry Pi version and OS edition:
 
-##### Raspberry Pi 1, Pi Zero, Pi Zero W, and Compute Module default build configuration
+##### If on the Raspberry Pi OS (32-bit)
+
+###### Raspberry Pi 1, Pi Zero, Pi Zero W, and Compute Module default build configuration
 
 ```bash
 cd linux
 KERNEL=kernel
 make bcmrpi_defconfig
 ```
 
-##### Raspberry Pi 2, Pi 3, Pi 3+, and Compute Module 3 default build configuration
+###### Raspberry Pi 2, Pi 3, Pi 3+, and Compute Module 3 default build configuration
 
 ```bash
 cd linux
 KERNEL=kernel7
 make bcm2709_defconfig
 ```
 
-##### Raspberry Pi 4 default build configuration
+###### Raspberry Pi 4 default build configuration
 
 ```bash
 cd linux
 KERNEL=kernel7l
 make bcm2711_defconfig
 ```
 
+##### If on the Raspberry Pi OS (64-bit)
+
+###### Raspberry Pi 3, Pi 3+, and Compute Module 3 default build configuration
+
+```bash
+cd linux
+KERNEL=kernel8
+make bcm2709_defconfig
```

Comment:
The cross-compiling instructions imply that this should be `bcmrpi3_defconfig` ?